### PR TITLE
chore(kuma-dp): remove envoy admin port flag

### LINKED ISSUE #4002: Remove deprecated admin port in kumactl arg
### Description

Admin port was specified with kumactl. This was deprecated and now it's a part of Dataplane object. We should remove it.

It was deprecated in Kuma 1.5.x

### PR DESCRIPTION
**Pull request by tharun208**
### Summary

remove deprecated `admin-port` command-line argument from `kuma-dp`

### Full changelog

* remove admin port config from kuma-cp and kubernetes side-car container
* removed deprecate messages and deleted files
* updated tests
### Issues resolved

Fix #4002

### Documentation

- [ ] Link to the website [documentation PR](https://github.com/kumahq/kuma-website/pull/XXX)

### Testing

- [ ] Unit tests
- [ ] E2E tests
- [ ] Manual testing on Universal
- [ ] Manual testing on Kubernetes

### Backwards compatibility

- [ ] Update [`UPGRADE.md`](/UPGRADE.md) with any steps users will need to take when upgrading.
- [ ] Add `backport-to-stable` label if the code follows our [backporting policy](/CONTRIBUTING.md#backporting)

### CODE REVIEW

**Comment by tharun208** [comment]
@jakubdyszkiewicz taking a look...

**Review by jakubdyszkiewicz — APPROVED** [review]

File: app/kuma-cp/cmd/run.go [hunk 1]
```diff
@@ -53,8 +53,6 @@ func newRunCmdWithOpts(opts kuma_cmd.RunCmdOpts) *cobra.Command {
 				return err
 			}
 
-			kuma_cp.PrintDeprecations(&cfg, cmd.OutOrStdout())
```
Comment (lobkovilya) [inline app/kuma-cp/cmd/run.go:56]: I don't think we should delete the actual calls to the `kuma_cp.PrintDeprecations`. We may need this mechanism in the future. You can simply make the `deprecations` list empty in `pkg/config/app/kuma-cp/deprecate.go`:
```go
var deprecations = []config.Deprecation{}
```

**Review by lobkovilya — CHANGES_REQUESTED** [review]

File: app/kuma-cp/cmd/run.go (hunk 1 above)
Comment (tharun208) [reply app/kuma-cp/cmd/run.go]: addressed

**Comment by lobkovilya** [comment]
Also, please take a look at https://github.com/kumahq/kuma/blob/master/pkg/xds/bootstrap/generator.go#L111-L118. And I believe we can delete `AdminPort` from the `BootstrapRequest`, right?

**Comment by tharun208** [comment]
> https://github.com/kumahq/kuma/blob/master/pkg/xds/bootstrap/generator.go#L111-L118

addressed 👍🏻 thank's for pointing out

**Comment by tharun208** [comment]
@lobkovilya is it good to merge?

**Comment by lobkovilya** [comment]
What about https://github.com/kumahq/kuma/blob/master/pkg/xds/bootstrap/types/bootstrap_request.go#L7? I think we don't need it

**Comment by tharun208** [comment]
> What about https://github.com/kumahq/kuma/blob/master/pkg/xds/bootstrap/types/bootstrap_request.go#L7? I think we don't need it

removed

**Comment by lobkovilya** [comment]
@Mergifyio update

**Review by lobkovilya — APPROVED** [review]

**Pull request merged**